plugins/hydra_rq_launcher/README.md

```diff
@@ -0,0 +1,67 @@
+# Hydra RQ Launcher
+
+Provides a [Redis Queue (RQ)](https://python-rq.org) launcher for Hydra:
+
+> RQ (Redis Queue) is a simple Python library for queueing jobs and processing them in the background with workers.
+
+
+### Installation
+
+This plugin requires Hydra 1.0 (Release candidate)
+
+```commandline
+pip install -e .
+```
+
+Usage of this plugin requires a Redis server.
+
+
+### Usage
+
+Once installed, add `hydra/launcher=rq` to your command line. Alternatively, override `hydra/launcher` in your config:
+
+```yaml
+defaults:
+  - hydra/launcher: rq
+```
+
+The default configuration is defined in `hydra_plugins/hydra_rq_launcher/config.py`. For a full description of the `enqueue` arguments, refer to the [documentation of Redis Queue](https://python-rq.org/docs/). Also note the [limitations of Redis Queue](https://python-rq.org/docs/#limitations).
+
+An example application using this launcher is provided in `plugins/hydra_rq_launcher/example`. Prerequisite is a running [Redis server](https://redis.io/topics/quickstart). 
+
+It is good practice not to store connection credentials in code, since they might be accidentally shared. Instead, the default configuration will read them from environment variables. 
+
+On a Linux system with bash, they can be set as follows:
+```commandline
+export REDIS_HOST="localhost"
+export REDIS_PORT="6379"
+export REDIS_DB="0"
+export REDIS_PASSWORD=""
+export REDIS_URL="redis://:$REDIS_PASSWORD@$REDIS_HOST:$REDIS_PORT/$REDIS_DB"
```

Comment:
mention that by default the plugin is using the environment variables (and document them).